src/views/Peers.tsx

```diff
@@ -154,99 +181,141 @@ export const Peers = () => {
         });
     }
 
+    const setUpdateGroupsVisible = (peerToAction:Peer, status:boolean) => {
+        if (status) {
+            dispatch(peerActions.setPeer({...peerToAction}))
+            dispatch(peerActions.setUpdateGroupsVisible(true))
+            return
```

Comment:
Codacy has a fix for the issue: [Replace `············return` with `······return;`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
      return;
```